src/components/shared/Extensions/PageTitle/PageTitle.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+
+import { PageTitleProps } from './types';
+import styles from './PageTitle.module.css';
+
+const displayName = 'Extensions.PageTitle';
+
+const MSG = defineMessages({
```

Comment:
move it to const file or use to translation  const { formatMessage } = useIntl(); and move text to json with english translactions